src/integrationTest/java/org/opentripplanner/base/IsochroneIT.java

```diff
@@ -0,0 +1,140 @@
+                .queryParam("cutoffSec", "900")
+                .request()
+                .header("Accept", "application/json")
+                .get();
+
+        FeatureCollection body = response.readEntity(FeatureCollection.class);
+
+        assertThat(response.getStatus(), equalTo(200));
+        assertIsochroneOk(body);
+    }
+
+    @Test
+    public void testTransitIsochrone() {
+        javax.ws.rs.core.Response response = target("/routers/bydgoszcz/isochrone")
+                .queryParam("fromPlace", "53.134802,17.991995")
+                .queryParam("mode", "WALK,TRANSIT")
+                .queryParam("disableRemainingWeightHeuristic", "true")
+                .queryParam("maxWalkDistance", "600")
+                .queryParam("softWalkLimit", "false")
+                .queryParam("time", "4:30pm")
+                .queryParam("date", "07-22-2020")
+                .queryParam("cutoffSec", "300")
+                .queryParam("cutoffSec", "600")
+                .queryParam("cutoffSec", "900")
+                .request()
+                .header("Accept", "application/json")
+                .get();
+
+        FeatureCollection body = response.readEntity(FeatureCollection.class);
+
+        assertThat(response.getStatus(), equalTo(200));
+        assertIsochroneOk(body);
+    }
+
+    @Test
+    public void testRentableVehiclesIsochrone() {
+        javax.ws.rs.core.Response response = target("/routers/bydgoszcz/isochrone")
+                .queryParam("fromPlace", "53.134802,17.991995")
+                .queryParam("mode", "WALK,CAR,BICYCLE")
+                .queryParam("startingMode", "WALK")
+                .queryParam("rentingAllowed", "true")
+                .queryParam("disableRemainingWeightHeuristic", "true")
+                .queryParam("maxWalkDistance", "600")
+                .queryParam("softWalkLimit", "false")
+                .queryParam("cutoffSec", "300")
+                .queryParam("cutoffSec", "600")
+                .queryParam("cutoffSec", "900")
+                .request()
+                .header("Accept", "application/json")
+                .get();
+
+        FeatureCollection body = response.readEntity(FeatureCollection.class);
+
+        assertThat(response.getStatus(), equalTo(200));
+        assertIsochroneOk(body);
+    }
+
+    @Test
+    public void testRentableVehiclesAndTransitIsochrone() {
+        javax.ws.rs.core.Response response = target("/routers/bydgoszcz/isochrone")
+                .queryParam("fromPlace", "53.134802,17.991995")
+                .queryParam("mode", "WALK,CAR,BICYCLE,TRANSIT")
+                .queryParam("startingMode", "WALK")
+                .queryParam("rentingAllowed", "true")
+                .queryParam("disableRemainingWeightHeuristic", "true")
+                .queryParam("maxWalkDistance", "600")
+                .queryParam("softWalkLimit", "false")
+                .queryParam("time", "4:30pm")
+                .queryParam("date", "07-22-2020")
```

Comment:
Haven't we deprecated `date`, `time` and prioritised using `timestampMillis`?

Reply:
fixed